File: go/design-patterns/facade/naive/callers_naive.go
// The living-room "Movie Night" button:
func movieNightButton() {
    amp := Amplifier{}; lights := Lights{}
    projector := Projector{}; box := StreamingBox{}

    lights.Dim(10)                 // ❌ caller must know all six APIs
    projector.On()
    projector.Wide()
    amp.On()
    amp.SetVolume(7)
    box.Play("Dune")               // …and the exact ORDER
}
// The mobile app and voice assistant repeat the same dance.

File: java/design-patterns/facade/naive/CallersNaive.java
// The living-room "Movie Night" button:
void movieNightButton() {
    Amplifier amp = new Amplifier();
    Lights lights = new Lights();
    Projector projector = new Projector();
    StreamingBox box = new StreamingBox();

    lights.dim(10);                 // ❌ caller must know all six APIs
    projector.on();
    projector.wide();
    amp.on();
    amp.setVolume(7);
    box.play("Dune");               // …and the exact ORDER
}
// The mobile app and voice assistant repeat the same dance.